job-server/src/main/scala/db/h2/migration/V0_7_4/V0_7_4__Migrate_Blobs.scala

```diff
@@ -0,0 +1,69 @@
+package db.h2.migration.V0_7_4
+
+import java.sql.Connection
+
+import scala.concurrent.Await
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.TimeoutException
+import scala.concurrent.duration.DurationInt
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration
+import org.slf4j.LoggerFactory
+
+import javax.sql.rowset.serial.SerialBlob
+import slick.dbio.DBIO
+import slick.driver.H2Driver.api.actionBasedSQLInterpolation
+import slick.jdbc.GetResult
+import slick.jdbc.PositionedParameters
+import slick.jdbc.SetParameter
+import spark.jobserver.slick.unmanaged.UnmanagedDatabase
+import java.sql.Blob
+import scala.util.control.NonFatal
```

Comment:
minor nit: please group the scala imports together, in the second section (lines 5-8)